ParserHooks.php

```diff
@@ -13,7 +13,7 @@
 	return 1;
 }
 
-define( 'ParserHooks_VERSION', '1.5.0' );
+define( 'ParserHooks_VERSION', '1.6.0 dev' );
```

Comment:
The bunny is grumpy and rather wants to define in semver style 1.6.0-alpha.

![55d26652b3143a728d23ba05e5d1b5da](https://cloud.githubusercontent.com/assets/1104078/24494935/dbcca3c8-1533-11e7-910b-4086e03c95b7.jpg)